src/vm/excep.cpp

```diff
@@ -7784,11 +7783,19 @@ LONG WINAPI CLRVectoredExceptionHandler(PEXCEPTION_POINTERS pExceptionInfo)
     //
     return retVal;
 #else // !FEATURE_PAL
+
+#if defined(WIN64EXCEPTIONS) && defined(FEATURE_HIJACK)
```

Comment:
It looks like moving the `#ifndef FEATURE_PAL` below the ` #endif // defined(WIN64EXCEPTIONS)`,  adding `defined(FEATURE_HIJACK)` to the `#if defined(WIN64EXCEPTIONS)` and removing the `#else` would do the trick.